frontend/packages/shared/src/contexts/ServicesContext.tsx

```diff
@@ -124,18 +122,13 @@ export const ServicesContextProvider = ({
   );
 
   return (
-    <ErrorBoundary
```

Comment:
I chose to remove this because errors are now caught by the RouteErrorBoundary instead. :)